app/views/games/show.html.erb

```diff
@@ -1 +1,19 @@
-<h1>Game page</h1>
+<br />
+<h1 class="text-center"><%= @game.name %></h1>
+<br />
+<table class="chess_table">
+  <% 7.downto(0).each do |row_number| %>
+    <tr>
+      <% 0.upto(7).each do |column_number| %>
+        <td class="<%= (row_number + column_number).even? ? "black_space" : "white_space"%>">
+          <% piece = @game.pieces.at(column_number, row_number) %>
+          <% if piece %>
+            <p class='piece'><%= piece.to_unicode %></p>
+          <% end %>
+        </td>
+      <% end %>
+    </tr>
+  <% end %>
+</table>
+<br />
+<br />
```

Comment:
There's code for the board, populate board and obstructed in this PR. 😄 

I bet what happened is you may have branched off of your existing branches (rather than master) when starting something new. Pulling your other changes along with it.

Ideally - you want each PR to only have the code in it that is for that change (so this would only have the obstructed code in it)